Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+MAINTAINER Shikher Verma "root@shikherverma.com"
+RUN apt-get update -y --fix-missing
+RUN apt-get install -y python python-pip python-dev build-essential git nodejs npm ruby ruby-dev
+# Install Dependencies
+RUN \
+  apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10 && \
+  echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' > /etc/apt/sources.list.d/mongodb.list && \
+  apt-get update && \
+  apt-get install -y mongodb-org && \
```

Comment:
I'd suggest to add --no-install-recommends to keep the final image minimal (and lower the build times). Using DEBIAN_FRONTEND=noninteractive also avoids getting stuck on configuration steps that require choices (which usually happens when you need to setup a password for a database or so).